test/stress_tests/test_actor_event_loop.py

```diff
@@ -0,0 +1,119 @@
+import ray
+import numpy as np
+import time
+from collections import Counter
+import json
+import sys
```

Comment:
Alphabetical order.

also, move `import ray` after the standard library imports (with a newline in between)